protocol/x/clob/keeper/process_operations.go

```diff
@@ -590,24 +590,6 @@ func (k Keeper) PersistMatchLiquidationToState(
 			)
 		}
 
-		// Stat fill amount in quote quantums as ratio of notional quote quantums
-		fillAmountToTotalSizeRat := new(big.Rat).Quo(
-			new(big.Rat).SetUint64(matchWithOrders.FillAmount.ToUint64()),
-			new(big.Rat).SetUint64(matchLiquidation.TotalSize),
-		)
-		filledQuoteQuantums := lib.BigRatRound(
-			new(big.Rat).Mul(
-				fillAmountToTotalSizeRat,
-				new(big.Rat).SetInt(absNotionalQuoteQuantums),
-			),
-			true,
-		)
-		telemetry.IncrCounterWithLabels(
-			[]string{types.ModuleName, metrics.LiquidationOrderNotionalQuoteQuantums, metrics.Filled, metrics.DeliverTx},
-			float32(filledQuoteQuantums.Uint64()),
-			matchLiquidation.GetMetricLabels(),
-		)
```

Comment:
this is being stat'ed in `process_single_match` which is shared between check tx and deliver tx